modules/bitgo/test/v2/unit/wallet.ts

```diff
@@ -2483,15 +2483,12 @@ describe('V2 Wallet:', function () {
               typedData,
               prv: 'secretKey',
             });
-            signedTypedData.should.deepEqual(txHash);
-            const actualArg = signTypedDataTssSpy.getCalls()[0].args[0];
-            const expected = '1901493fbf2ae1c27c3ced26a89070c6ab5d3fbf37ed778de9378e7703b7d1f116b3883077a61826129b98b622e54fc68c5008d1b1c16552e1eda6916f870d719220';
-            actualArg.typedData.typedDataEncoded.toString('hex').should.equal(expected);
+            signedTypedData.should.deepEqual(expected);
           });
 
           it('should sign typed data V4 when custodianMessageID is provided', async function () {
             typedData.txRequestId = txRequestId;
-            const signTypedDataTssSpy = sinon.spy(tssEthWallet, 'signTypedDataTss' as any);
+            sinon.spy(tssEthWallet, 'signTypedDataTss' as any);
```

Comment:
Is this used anymore in test?, If not kindly remove this

Reply:
I think you are looking at a stale version of the PR ([latest version](https://github.com/BitGo/BitGoJS/blob/a90f86e57c27e4e66bbaee705bfddd0b4d1c6e21/modules/bitgo/test/v2/unit/wallet.ts#L2491))